packages/manager-react-components/src/components/datagrid/datagrid.spec.tsx

```diff
@@ -220,3 +221,53 @@ it('should disable overflow of table', async () => {
   );
   expect(container.querySelectorAll('.overflow-hidden').length).toBe(1);
 });
+
+it('should have the default number of loading row when isLoading is true and numberOfLoadingRows is not specified', async () => {
+  const { queryAllByTestId } = render(
+    <Datagrid columns={sampleColumns} items={[]} totalItems={0} isLoading />,
+  );
+  expect(queryAllByTestId('loading-row')?.length).toBe(
```

Comment:
```suggestion
  expect(queryAllByTestId('loading-row').length).toBe(
```
queryAll always returns an array isn't it ?

Reply:
you are right, for queryAll it's always returning an array, modified !